landscape_setup/concourse.py

```diff
@@ -111,6 +111,38 @@ def create_image_pull_secret(
             image_pull_secret_name=image_pull_secret_name
         )
 
+MITM_CONFIG_CONFIGMAP_NAME = 'mitm-config'
+MITM_SCRIPTS_CONFIGMAP_NAME = 'mitm-scripts'
+MITM_SCRIPT_DIR = '/.mitmscripts'
+MITM_CONFIG_DIR = '/.mitmproxy'
+
+
+@ensure_annotations
+def create_proxy_configmaps(
+    proxy_cfg: ProxyConfig,
+    namespace: str,
+):
+    """Create the config map that contains the configuration of the mitm-proxy"""
+    not_none(proxy_cfg)
```

Comment:
no need to also check for non-none-ness - this is already done by @ensure_annotations